firewood/src/storage/buffer.rs

```diff
@@ -484,8 +484,8 @@ async fn process(
                 wal_cfg.max_revisions,
                 final_path.as_path(),
             )
-            .await
-            .unwrap_or_else(panic_on_intialization_failure_with(&rootpath, &waldir));
+                .await
```

Comment:
All the code in this repo is formatted using cargo fmt on the latest stable. We have some checks to ensure that it stays that way.

Not sure what you're doing to get this alternate indentation; perhaps not the latest version of rustfmt?

Reply:
Thanks, it seems my intellij was using its own internal formatter and not `rustfmt` as I thought

I've also updated my rustfmt to the latest.

```
$ rustfmt --version
rustfmt 1.6.0-stable (d5c2e9c3 2023-09-13)
```

Updating pull request with all the changes requested in a few mins...